### PR TITLE
feat: add cfms v1.5 features and move TCAS out of exp.md

### LINKED ISSUE #454: Move TCAS out of experimental page
Since TCAS is in dev should move relevant information to feature guides OR advanced guides.

### LINKED ISSUE #458: Add CFMS v1.5 to cfms.md
As requested by dev team

### PR DESCRIPTION
**Pull request by Valastiri**
fixes #458 
fixes #454 

## Summary
- Use information provided by https://github.com/flybywiresim/a32nx/pull/6334 to build out a feature list on the cFMS page.
- Split the feature list by using admonitions with `Version 1.5` already open when viewing the page. Version 1 starts collapsed.
- TCAS information has been moved to the flight guidance in `pilot's corner`.
- Mention an upcoming guide for TCAS on the cFMS page.
### Location

- docs/fbw-a32nx/feature-guides/cFMS.md
- docs/fbw-a32nx/support/exp.md
- docs/pilots-corner/advanced-guides/flight-guidance/tcas.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

**Comment by aguther** [comment]
LGTM

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 1]
```diff
@@ -14,32 +14,61 @@ For guides on utilizing features included with our custom FMS see the [Guides an
...
 ## Guides and Information
 
 - [MCDU Fix Info](../../pilots-corner/advanced-guides/flight-planning/fixinfo.md)
 - [Flight Plan Discontinuities](../../pilots-corner/beginner-guide/preparing-mcdu.md#discontinuity)
+- TCAS Guide - <span style="color:red">**Advanced Guide Coming Soon**</span>
 
 ## Known Issues
 
-- TCAS is not implemented yet. No traffic shown, TCAS elements are cosmetic only.
+- CA leg terminations are sometimes in the wrong place, and do not adapt to V/S.
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: Not sure what that means? "do not adapt to V/S"

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 2]
```diff
@@ -14,32 +14,61 @@ For guides on utilizing features included with our custom FMS see the [Guides an
...
+    - Roll anticipation distance.
+    - Improved flight plan rendering / drawing.
+    - Improved ND display filters for ARPT, VOR, NDB, WPTs.
+    - MCDU FIX INFO page for radials and distance circles on navigation fixes. See [Guides and Information](#guides-and-information) below.
 
 ## Guides and Information
 
 - [MCDU Fix Info](../../pilots-corner/advanced-guides/flight-planning/fixinfo.md)
 - [Flight Plan Discontinuities](../../pilots-corner/beginner-guide/preparing-mcdu.md#discontinuity)
+- TCAS Guide - <span style="color:red">**Advanced Guide Coming Soon**</span>
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: Wouldn't do this in red - not sure when we get around to it.

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 3]
```diff
@@ -14,32 +14,61 @@ For guides on utilizing features included with our custom FMS see the [Guides an
...
-    - [x] TF: Track to a Fix defines a great circle track over ground between two known databases fixes.
-    - [x] RF: Constant Radius Arc defines a constant radius turn between two database fixes, lines tangent to the arc and a center fix.
-    - [x] VM: Heading to a manual termination defines a specified heading until a Manual termination.
-- [x] Cross-track error indicator on the ND.
-- [x] Improved LNAV turn prediction.
-- [x] Roll anticipation distance.
-- [x] Improved flight plan rendering / drawing.
-- [x] Improved ND display filters for ARPT, VOR, NDB, WPTs.
-- [x] MCDU FIX INFO page for radials and distance circles on navigation fixes. See [Guides and Information](#guides-and-information) below.
+???+ info "Version 1.5"
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: Should probably have normal font size for these admons - it is rather small like this

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 4]
```diff
@@ -14,32 +14,61 @@ For guides on utilizing features included with our custom FMS see the [Guides an
...
 
 ## Known Issues
 
-- TCAS is not implemented yet. No traffic shown, TCAS elements are cosmetic only.
+- CA leg terminations are sometimes in the wrong place, and do not adapt to V/S.
+- Some path captures will be incorrectly drawn. This will not affect guidance.
+- INTCPT calculation can be off on large distances.
+- Course captures do not adapt to PPOS when nextLeg is active. This can cause the final path to be off to the side.
 - WX and TERR on ND are not implemented yet. We are waiting for better API support by Microsoft Flight Simulator. See our [Forums Feature Request](https://forums.flightsimulator.com/t/implement-weather-and-terrain-api-s-for-aircraft-developers-to-implement-accurate-radar-predictive-windshear-egpws-and-metar-wind-uplink/442016){target=new}.
 - Only 3 Leg Types/1 Transition supported at this time. Although these cover most of the important types there might be some issues with other leg types during flight guidance. We will be significantly improving this in future updates.
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md]: Needs update or removal

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 5]
```diff
@@ -14,32 +14,61 @@ For guides on utilizing features included with our custom FMS see the [Guides an
...
 - Rendering of flight path on the ND of terminal procedure legs may be glitched or incorrect during cruise. - [See Special Notes](../feature-guides/cFMS.md#flight-path-rendering).
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md:75]: Are these and the ones below still Known Issues?

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/fbw-a32nx/feature-guides/cFMS.md (hunk 2 above)
Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/cFMS.md]: Will adjust this to just WIP instead and remove the span

File: docs/fbw-a32nx/feature-guides/cFMS.md (hunk 1 above)
Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/cFMS.md]: swapping this to "does not adapt"

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 6]
```diff
@@ -39,7 +65,10 @@ For guides on utilizing features included with our custom FMS see the [Guides an
 
 ## Known Issues
 
-- TCAS is not implemented yet. No traffic shown, TCAS elements are cosmetic only.
+- CA leg terminations are sometimes in the wrong place, and does not adapt to V/S.
+- Some path captures will be incorrectly drawn. This will not affect guidance.
+- INTCPT calculation can be off on large distances.
+- Course captures do not adapt to PPOS when nextLeg is active. This can cause the final path to be off to the side.
 - WX and TERR on ND are not implemented yet. We are waiting for better API support by Microsoft Flight Simulator. See our [Forums Feature Request](https://forums.flightsimulator.com/t/implement-weather-and-terrain-api-s-for-aircraft-developers-to-implement-accurate-radar-predictive-windshear-egpws-and-metar-wind-uplink/442016){target=new}.
 - Only 3 Leg Types/1 Transition supported at this time. Although these cover most of the important types there might be some issues with other leg types during flight guidance. We will be significantly improving this in future updates.
```
Comment (frankkopp) [inline docs/fbw-a32nx/feature-guides/cFMS.md]:
```suggestion
```

File: docs/fbw-a32nx/support/exp.md [hunk 7]
```diff
@@ -36,24 +36,11 @@ We have made some improvements to the AP and FBW. We **highly recommend** using
...
 
-### TCAS Specific Issues
+### TCAS
 
-- No support for offline AI traffic (sim limitation)
-- No support of multiplayer (MSFS) traffic (sim limitation)
-- Possible false detection of ground traffic
-- Possible ghost TA / RA due to jumping traffic (*improvements and changes expected*)
-- No performance optimization yet
+!!! tip "Now Available in the Development Version"
```
Comment (frankkopp) [inline docs/fbw-a32nx/support/exp.md:41]: Do we need this? Would need to remember to remove it in the near future :)

**Review by frankkopp — APPROVED** [review]

**Comment by Watsi01** [comment]
For the docs: 
with the line
` - No support of multiplayer (MSFS) traffic (sim limitation)`
I would add something like 
`support for online network traffic (Vatsim, etc)`
Cause people get confused (I recognized)

File: docs/fbw-a32nx/support/exp.md (hunk 7 above)
Comment (Valastiri) [reply docs/fbw-a32nx/support/exp.md:41]: Yea just temporarily -- technically we'd change the whole exp page anyways once complete. It's more for posterity to move the contents.

**Pull request merged**